.github/workflows/smoke.yml

```diff
@@ -70,4 +70,4 @@ jobs:
         uses: actions/upload-artifact@v1.0.0
         with:
           name: smoke-test-logs
```

Comment:
@obabec  Should the name be smoke-test-logs.zip as well to reflect what content to expect?

Reply:
Name of artifact is just name what github shows you in downloadable assets. Check last build, everything end up nicely.